src/tests/test_volunteer_profile.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from django.urls import reverse
+from rest_framework import status
+
+
+@pytest.mark.django_db
+class TestVolunteerProfile:
+
+    url = reverse("profile")
+
+    def test_get_volunteer_profile(self, user_client):
+        response = user_client.get(self.url)
+        assert response.status_code == status.HTTP_200_OK
+
+    def test_patch_volunteer_profile(self, user, user_client):
+        user_client.force_authenticate(user=user)
+        data = {
+            "call_sign": "Test Sign",
+            "birth_date": "2000-01-01",
+        }
+        response = user_client.patch(self.url, data)
+        assert response.status_code == status.HTTP_200_OK
+
+        user.refresh_from_db()
+        assert user.volunteer.call_sign == "Test Sign"
```

Comment:
```suggestion
        assert user.volunteer.call_sign == data["call_sign"]
```
Удобно, если поменяли тестовые данные, то и поменяются данные проверки.